src/garage/torch/algos/dqn.py

```diff
@@ -0,0 +1,286 @@
+"""This modules creates a DDPG model in PyTorch."""
+import collections
+import copy
+
+from dowel import logger, tabular
+import numpy as np
+import torch
+import torch.nn.functional as F
+
+from garage import _Default, log_performance, make_optimizer
+from garage._functions import obtain_evaluation_episodes
+from garage.np.algos import RLAlgorithm
+from garage.sampler import FragmentWorker
+from garage.torch import dict_np_to_torch, global_device
+
+
+class DQN(RLAlgorithm):
```

Comment:
does this match the hparams used by TF DQN?

Reply:
It does yes, with a couple of extra ones for evaluation.